automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -28,169 +30,468 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
 
   "A workspace with an authorization domain" - {
+    "with one group inside of it" - {
+      "can be created" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec") { authDomainName =>
+          withCleanUp {
+            val workspaceListPage = signIn(Config.Users.fred)
 
-    "can be created by a user who is in a managed group" in withWebDriver { implicit driver =>
-      withGroup("AuthDomainSpec") { authDomainName =>
-        withCleanUp {
-          val workspaceListPage = signIn(Config.Users.fred)
-
-          val workspaceName = "AuthDomainSpec_create_" + randomUuid
-          register cleanUp api.workspaces.delete(projectName, workspaceName)
-          val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Option(authDomainName)).awaitLoaded()
+            val workspaceName = "AuthDomainSpec_create_" + randomUuid
+            register cleanUp api.workspaces.delete(projectName, workspaceName)
+            val workspaceDetailPage = workspaceListPage.createWorkspace(projectName, workspaceName, Set(authDomainName)).awaitLoaded()
 
-          workspaceDetailPage.ui.readAuthDomainGroups should include (authDomainName)
+            workspaceDetailPage.ui.readAuthDomainGroups should include(authDomainName)
 
-          workspaceListPage.open.filter(workspaceName)
-          workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
+            workspaceListPage.open.filter(workspaceName)
+            workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
```

Comment:
i put these two cases in the same test for the sake of speed. since they're both cosmetic assertions, it seemed wasteful to create and tear down a workspace to test something cosmetic that was already visible to the test. but maybe that's not best-practice. 

i'm happy to break them apart if we want this level of granularity and don't care about the impact on speed